docs/shared-content-source/docs/modules/develop/pages/debugging-remote.adoc

```diff
@@ -0,0 +1,41 @@
+:page-partial:
+
+include::ROOT:partial$include.adoc[]
+
+
+Debugging a remote application is possible just by applying the https://docs.oracle.com/javase/8/docs/technotes/guides/jpda/conninv.html#Invocation[standard] configuration to debugging a remote JVM. That is, adding this conf to JAVA_OPTS through our xref:cloudflow-configuration.adoc[configuration model]. 
+	
+	-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005
+
+
+NOTE: the address can NOT be changed, it shall remain 5005.
```

Comment:
we can bind the address to an sbt `settingKey`, if a user wants to debug multiple applications or in different environments, this sounds like a limitation, WDYT?